src/i_oalsound.c

```diff
@@ -521,6 +605,7 @@ boolean I_OAL_InitSound(int snd_module)
         (alIsExtensionPresent("AL_EXT_SOURCE_RADIUS") == AL_TRUE);
     InitDeferred();
     ResetParams();
+    SetEqualizer();
```

Comment:
Does this still work after switching "modules" which would trigger a reinit (see below).

Reply:
I can't do any more checks today. Just leaving the code here. 😛